src/configure.py

```diff
@@ -219,6 +196,48 @@ def print_help( self, *args, **kwargs ):
         if "print_detail" in kwargs: self.print_option()
         if "epilog"       in self.program: print(self.program["epilog"])
 
+class SystemSetting( dict ):
+    """
+    Store the system settings from the default setting file.
+
+    Format of the setting file:
+    1. Comment starts with `#`.
+    2. The line begins with the variable name, followed by one or multiple spaces, and then the value.
+    3. Only the fisrt value of the line will be loaded.
+    4. If a variable is defined multiple times, only the last occurrence will be used.
+    """
+    def __init__( self, *args, **kwargs ):
+        super().__init__( *args, **kwargs )
+
+    def get_default( self, key, default_val ):
+        return self.get(key, default_val)
+    
+    def load( self, pathname ):
+        """Load the system settings from the default setting file. If a setting exists,
+        it will be overwritten. Return `False` if the file does not exist.
+
+        Parameters:
+            pathname : str - The path of the default setting file to be loaded.
+
+        Returns:
+            bool - Whether the file exists.    
```

Comment:
Remove trailing blanks.
```suggestion
            bool - Whether the file exists.
```